packages/modal/src/Modal.styled.js

```diff
@@ -36,24 +36,34 @@ export const ConstrainedCard = styled(Card)`
   position: relative;
   display: flex;
   flex-direction: column;
+  border: none;
+  overflow: hidden;
+
+  .closeButton {
+    position: absolute;
+    top: 16px;
+    right: 16px;
+    font-size: 32px;
+    color: ${colors.GREY_16};
+    &:hover {
+      color: ${colors.GREY_32};
+    }
+  }
 `;
 
 export const ContentWrapper = styled.div`
   flex: 1;
   overflow: auto;
   -webkit-overflow-scrolling: touch;
+  padding: 16px;
 `;
 
-export const Header = styled.div`
+export const Section = styled.div`
```

Comment:
We could use the `header` tag on Header and `footer` on Footer

Reply:
Just the `header` tag fits, `footer` is used to display information about the content. I thinking that, as this footer will be used all the times to display actions, we can name it  `ActionBar`. What do you think?